src/routes/news/components/NewsListItem.js

```diff
@@ -29,22 +31,27 @@ type ListItemViewPropsType = {|
   theme: ThemeType
 |}
 
-const ListItemView: StyledComponent<
-  ListItemViewPropsType,
-  ThemeType,
-  *
-> = styled.View`
+const ListItemView: StyledComponent<ListItemViewPropsType, ThemeType, *> = styled.View`
   flex-direction: ${props => contentDirection(props.language)};
 `
 
-const ReadMoreWrapper: StyledComponent<
-  { language: string, children: React.Node },
-  {},
-  *
-> = styled.View`
+const ReadMoreWrapper: StyledComponent<{ language: string, children: React.Node }, {}, *> = styled.View`
   flex-direction: ${props => contentDirection(props.language)};
   justify-content: flex-end;
   width: 100%;
+  align-self: center;
+`
+
+const Icon: StyledComponent<{ isTunews: ?boolean }, ThemeType, *> = styled(MaterialIcon)`
+  font-size: 20px;
+  color: black;
+  top: 4px;
+  right: 5px;
+  left: 0px;
```

Comment:
Is this correctly handled in LTR/RTL layouts?

Edit: AFAICS correctly handled using transform in render()

Reply:
AFAICT, it is looking good in UI review :D